### PR TITLE
Enable Categorisation of CDDP DSDM into subcategories

### PR DESCRIPTION
**Pull request by slubwama**

### CODE REVIEW

File: omod/src/main/resources/config.xml [hunk 1]
```diff
@@ -257,6 +257,13 @@
         <description>The UUID of the Reception</description>
     </globalProperty>
 
+    <globalProperty>
+        <property>ugandaemr.enableCDDPCategorization</property>
```
Comment (dbaluku) [inline omod/src/main/resources/config.xml:261]: @slubwama I think we should name this to enableDSDModelSubCategorization such that its not specific to CDDP only

File: omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml [hunk 2]
```diff
@@ -2664,6 +2726,34 @@
...
+                                                    <div class="col-md-3">
+                                                        <label>Pharmacy</label>
+                                                        <div class="form-group">
+                                                            <select class="form-control" id="pharmacy_selection_id" name="pharmacy_selection_id">
+                                                                <option value="">Select Pharmacy</option>
+                                                            </select>
+                                                        </div>
+                                                    </div>
+                                                    <div class="col-md-3">
+                                                        <obs id="pharmacy_id" conceptId="166476"
```
Comment (dbaluku) [inline omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml:2746]: What is the right concept for this field

File: omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml [hunk 3]
```diff
@@ -2664,6 +2726,34 @@
...
+                                                             labelText="Pharmacy Code"/>
+                                                    </div>
+                                                    <div class="col-md-3">
+                                                        <obs id="clddp_id" conceptId="166475"
```
Comment (dbaluku) [inline omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml:2750]: Concept for this is also reflecting "Do you have any complaints today?"

File: api/src/main/resources/metadata/Custom_Concepts.xml [hunk 4]
```diff
@@ -823,6 +823,15 @@
     <concept concept_id="165139" retired="false" datatype_id="4" class_id="18" is_set="false" creator="1" date_created="2018-09-28 11:58:02"   uuid="73313f20-c321-11e8-a355-529269fb1459"/>
     <concept concept_id="165138" retired="false" datatype_id="4" class_id="18" is_set="false" creator="1" date_created="2018-09-28 11:56:02"   uuid="733144c0-c321-11e8-a355-529269fb1459"/>
```
Comment (dbaluku) [inline api/src/main/resources/metadata/Custom_Concepts.xml:825]: These concepts are already existing

**Comment by slubwama** [comment]
Another PR for this By Bdaluku https://github.com/METS-Programme/openmrs-module-ugandaemr/pull/101

**Pull request closed without merging**